### ISSUE TITLE
build error MSB3721 

### ISSUE THREAD

**Issue by Zuo1ji** [AlexeyAB/darknet#1969]
hello,

i have met some error

build on 
MSVC2013 update 5 (12.0.41012.0)
cuda 9.1
cudnn 7.0
opencv 300

and yes,i install MSVS2013 then i install CUDA 9.1
and, i have set Toolset to VS2013(v120)
all things have been done, but i got,

Error	223	error : expected a ";"	darknet\src\im2col_kernels.cu	853	1	yolo_cpp_dll

and

Error	352	error MSB3721: The command ""C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v9.1\bin\nvcc.exe" -gencode=arch=compute_30,code=\"sm_30,compute_30\" -gencode=arch=compute_52,code=\"sm_52,compute_52\" --use-local-env --cl-version 2013 -ccbin "F:\software\visual studio 2013 community\VC\bin\x86_amd64" -x cu  -IC:\opencv_3.0\opencv\build\include -I..\..\3rdparty\include -I"C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v9.1\include" -I"C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v9.1\include" -I\include -I"C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v9.1\include"     --keep-dir x64\Release -maxrregcount=0  --machine 64 --compile -cudart static     -DCUDNN -DGPU -DOPENCV -DYOLODLL_EXPORTS -D_TIMESPEC_DEFINED -D_SCL_SECURE_NO_WARNINGS -D_CRT_SECURE_NO_WARNINGS -DWIN32 -DNDEBUG -D_CONSOLE -D_LIB -D_WINDLL -D_MBCS -Xcompiler "/EHsc /W3 /nologo /O2 /FS /Zi  /MD " -o x64\DLL_Release\im2col_kernels.cu.obj "darknet\src\im2col_kernels.cu"" exited with code 1.	C:\Program Files (x86)\MSBuild\Microsoft.Cpp\v4.0\V120\BuildCustomizations\CUDA 9.1.targets	707	9	yolo_cpp_dll

i have searched some question before, but got no solution.

but now, i have no idea about this problem

**Comment by SuperArwenAntoinette** [comment]
I only saw 2015 for the MSVS in the whole execution of this, are you doing this for the custom build?

**Comment by Zuo1ji** [comment]
Yes, i want to use it as dll on my another solution which is build on MSVS 2013

**Comment by Adesun** [comment]
after opening vs2013 command prompt,  running command just as above and see the message displayed.

**Comment by JiaojiaoHan** [comment]
im2col_kernels.cu     line 290    "return ~(a^b) & 0b1;"  ->  "return ~(a^b) & 0x1;"
gemm.c   line 340   "return (lookup[n & 0b1111] << 4) | lookup[n >> 4];"  ->  "return (lookup[n & 0x1111] << 4) | lookup[n >> 4];"
